cpp1st/week01/taehwan/LambdaExpression.cpp

```diff
@@ -14,6 +14,30 @@ class LambdaExample {
     void goTest();
     void test1();
     void test2();
+    void test3();
+    void test4();
+};
+
+/// test class for example
+class testClassForTest3 {
+public:
+    testClassForTest3() {
+        array = {60, 20, 30, 80, 50};
+    }
+
+    void Print() {
+        for(auto &n : array)
+            cout << n << '\t';
+
+        cout << endl;
+    }
+
+    void Sort(function<int(int, int)> cmp) {
+        sort(array.begin(), array.end(), cmp);
+    }
+
+private:
+    array<int, 5> array;
 };
 
 /// test function for example
```

Comment:
consider to add const keyword for char*.

Reply:
The correct location is line 44. I couldn't place my comment into exact line. because there is no change in that line.

Reply:
I've updated.